### ISSUE TITLE
Embedded newlines make the PlantUML source invalid

### ISSUE THREAD

**Issue by hindsholm** [pmorch/c4viz#2]
Embedded newlines like
```
myUI = container "Xsys UI\nsomething" "Provides a UI for..." "SPA"
```
cause the generated PlantUML to break.

Otherwise a nice utility!

**Comment by pmorch** [comment]
Hey! My first issue from the field... Do you have a full DSL file I can test with?

**Comment by hindsholm** [comment]
Glad to be the first :smile: 

Here is a DSL for you.
[workspace.txt](https://github.com/pmorch/c4viz/files/7750546/workspace.txt)

**Comment by pmorch** [comment]
Simon, the author and maintainer of Structurizr, the main dependency for this project has already fixed this issue upstream.

Thanks, Simon! 

When a release has been made of the upstream project on [maven central](https://search.maven.org/search?q=a:structurizr-export), I'll update the dependency version here and re-release. I've set a reminder for myself to check again on Jan 5, 2022.

**Comment by pmorch** [comment]
I've just released version 0.2.0 that uses com.structurizr:structurizr-export:1.3.0 where this is fixed.
